docs/docs/guides/build/assets/asset-selection-syntax.md

```diff
@@ -1,58 +1,42 @@
-| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `ASSET_KEY`           | Selects a single asset by asset key. [See an example](#single-asset).                                                                                                                                                                                                                                                                                       |
-| `COMPONENT/COMPONENT` | Selects an asset key with multiple components, such as a prefix, where slashes (`/`) are inserted between components. [See an example](#multiple-key-components).                    |
-| `*ASSET_KEY`          | Selects an asset and all of its upstream dependencies. [See an example](#all-upstream).                                                                                                                                                                                                                             |
-| `ASSET_KEY*`          | Selects an asset and all of its downstream dependencies. [See an example](#all-downstream).                                                                                                                                                                                                                           |
-| `+ASSET_KEY`          | Selects an asset and one layer upstream of the asset. Including multiple `+`s will select that number of upstream layers from the asset. Any number of `+`s is supported. [See an example](#specific-upstream).    |
-| `ASSET_KEY+`          | Selects an asset and one layer downstream of the asset. Including multiple `+`s will select that number of downstream layers from the asset. Any number of `+`s is supported. [See an example](#specific-downstream). |
+| Asset selection | Example | Description |
+| --- | --- | --- |
+| **Specific asset** | `key:"my_asset"` | Selects assets with key `my_key`. |
+| **Entire graph** | `*` | Displays all assets and connections. |
 
-## Examples
+### Select a single asset \{#single-asset}
 
-The examples in this section use the following asset graph from the [Dagster University Essentials project](https://github.com/dagster-io/project-dagster-university) to demonstrate how to use the selection syntax:
-
-![Screenshot of Daggy U project graph](/images/guides/build/assets/asset-selection-syntax/asset-selection-syntax-dag.png)
-
```

Comment:
Should we use the Daggy U project graph as an example?

Reply:
I was wondering that myself -- I think Daggy U is on dagster v1.9, so the project graph would be OK, but the asset selection interface wouldn't have the new syntax. We might need different examples for the ones that were carried over from the old asset selection syntax doc that used Daggy U / taxi trips data.